operator/duties/fetcher.go

```diff
@@ -106,14 +106,17 @@ func (df *dutyFetcher) updateDutiesFromBeacon(logger *zap.Logger, slot phase0.Sl
 	if len(duties) == 0 {
 		return nil
 	}
-	// print the newly fetched duties
-	var toPrint []serializedDuty
-	for _, d := range duties {
-		toPrint = append(toPrint, toSerialized(d))
+
+	var b strings.Builder
```

Comment:
I would note here that if we need to format a duty id list in other places we should move this functionality to `fields.go`. 
@moshe-blox lets merge this!

Reply:
agreed, merging..